app.js

```diff
@@ -7,7 +7,13 @@ import adminMiddleware from './src/middleware/adminMiddleware';
 import apiMiddleware from './src/middleware/apiAuth';
 import adminRoutes from './src/routes/adminRoutes';
 
-dotenv.config();
+if (process.env.NODE_ENVIRONMENT === 'prod') {
```

Comment:
Optional but it would be better if you use just `NODE_ENV`

Reply:
Okay

Reply:
Okay